cpu/stm32/periph/eth.c

```diff
@@ -435,8 +512,19 @@ void stm32_eth_isr_eth_wkup(void)
     cortexm_isr_end();
 }
 
+int stm32_eth_get_rx_status_owned(void)
```

Comment:
should this be `static`?

Reply:
This function is not needed (see comment below). Please drop it again.